### ISSUE TITLE
inspector protocol: params.args[0].value.match is not a function

### ISSUE THREAD

**Issue by pflannery** [Microsoft/vscode-node-debug2#107]
```js
// index.js
console.log(123)
```

```js
// launch.json
    {
      "type": "node",
      "request": "launch",
      "name": "Launch Program",
      "program": "${workspaceRoot}/index.js",
      "protocol": "inspector"
    },
```

line [317](https://github.com/Microsoft/vscode-node-debug2/blob/master/src/nodeDebugAdapter.ts#L317) causes the following error
```js
TypeError: params.args[0].value.match is not a function
    at NodeDebugAdapter.onConsoleAPICalled (d:\dev\github\microsoft\vscode-node-debug2\src\nodeDebugAdapter.ts:317:109)
```

I see this error in vscode v1.12.2 and vscode v1.13.0-insider
I also downloaded this repo as master at commit 4543b147925d6cc98e0dc3f669d5509bcd2513a8 and still see this error.

**Comment by roblourens** [comment]
Is there a message that you expect to be logged right before this? Or can you set `"trace": true` and upload the log here?

**Comment by roblourens** [comment]
Are you sure it's exactly the same in 12.2? That line of code didn't exist in 12.2.

**Comment by roblourens** [comment]
I can repro. Dumb mistake, really needs to be fixed. Not in 12.2.

**Comment by roblourens** [comment]
Fixed in a926fb9c815a45a0201b2fa15f78462f1052e987
